examples/samples/topicsGettingStarted.ts

```diff
@@ -0,0 +1,96 @@
+import {
+  OnMessage,
+  OnError,
+  MessagingError,
+  delay,
+  ServiceBusMessage,
+  generateUuid,
+  Namespace,
+  SendableMessageInfo,
+  SubscriptionClient,
+  TopicClient
+} from "../../lib";
+import * as dotenv from "dotenv";
+dotenv.config();
+
+const str = process.env.SERVICEBUS_CONNECTION_STRING || "";
+const topic = process.env.TOPIC_NAME || "";
+const subscription1 = process.env.SUBSCRIPTION_NAME1 || "";
+const subscription2 = process.env.SUBSCRIPTION_NAME2 || "";
+const subscription3 = process.env.SUBSCRIPTION_NAME3 || "";
+
+console.log("str: ", str);
+console.log("path: ", topic);
+
+let ns: Namespace;
+async function main(): Promise<void> {
+  ns = Namespace.createFromConnectionString(str);
+  const client = ns.createTopicClient(topic);
+
+  const subscription1Client = ns.createSubscriptionClient(topic, subscription1);
+  const subscription2Client = ns.createSubscriptionClient(topic, subscription2);
+  const subscription3Client = ns.createSubscriptionClient(topic, subscription3);
+
+  //retrieving message one by one from each subscription
+  receiveMessage(subscription1Client, "color:red;");
+  receiveMessage(subscription2Client, "color:green;");
+  receiveMessage(subscription3Client, "color:cyan;");
```

Comment:
What is the purpose of these colors?

Reply:
We are retrieving messages one by one from  each subscription and passing color variable to differentiate the subscription data(just like .Net sample). See the output and Please let me know if I need to change anything.
<img width="685" alt="topicgettingstarted" src="https://user-images.githubusercontent.com/45216704/50118923-152f6e00-0206-11e9-96a9-a3842cd8571d.png">

Reply:
`client.name`